### PR TITLE
account.fetch() and account.profile.fetch() fixies

### PR DESCRIPTION
**Pull request by gr2m**
see hoodiehq/hoodie-account#37 & hoodiehq/hoodie-account#39

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,11 +27,11 @@ function Account (options) {
     isSignedIn: require('./lib/is-signed-in').bind(null, state),
     hasInvalidSession: require('./lib/has-invalid-session').bind(null, state),
     get: require('./lib/get').bind(null, state),
-    fetch: require('./lib/fetch').bind(null, state, 'account'),
+    fetch: require('./lib/fetch').bind(null, state),
     update: require('./lib/update').bind(null, state),
     profile: {
       get: require('./lib/profile-get').bind(null, state),
-      fetch: require('./lib/profile-fetch').bind(null, state, 'account.profile'),
```
Comment (gr2m) [inline index.js]: these were remains of how a former implementation, now we have separate methods `fetch` and `profile-fetch`

**Comment by gr2m** [comment]
I run the `@hoodie/account` tests prepared for hoodiehq/hoodie-account#37 & hoodiehq/hoodie-account#39 locally and this PR fixes the issue.

Ping @hoodiehq/maintainers 👀

**Comment by HipsterBrown** [comment]
LGTM :+1:

**Comment by gr2m** [comment]
LGTM

**Pull request merged**